_routes/posts.md

```diff
@@ -172,6 +172,12 @@ GET /api/v1/posts/:post_guid
 }
 ~~~
 
+### Errors
+
+Status code  |  Error reason
+-|-
+404  |  Post with provided guid was not found
```

Comment:
404 can also mean that you're not allowed to see the post. Probably not just for this route. I wonder whether we should document that fact or not.

Reply:
Maybe we can add another column in the table for the "Errors" page with additional notes and put this information there.